### DISCUSSION TITLE
Hi all just to show how my move to Axmol Engine went.

### DISCUSSION THREAD

**Discussion by GamesStudioTwo** [axmolengine/axmol#1912] [Show and tell]
Hi all, Just ported old stuff to axmol engine. Here  is my little show and tell. every thing has gone perfect.
Big big thanks to all those making it happen when I have fully finished I will post more demos. i will be giving away my joystick and debug to help people wanting to create for mobile soon. So Enjoy.

https://www.youtube.com/watch?v=arbdwd84NEQ

**Comment by paulocoutinhox** [comment]
Very nice, can you share the source code?

**Reply by GamesStudioTwo** [reply]
I'm going to be doing a tut on the hole engine so people can build there Owen.
but I'm about to do a another video a more deep dive on what I have created as the first video was just me saying I have ported over but it dose not really reflect the engine at it's best because the emulation was just so bad. So new video coming soon. hopefully ill have it up tonight for a more in-depth view or what I have done so far. I wont be releasing the source for this but I do have another project that i will be it will crate code for your projects.  but I need to port that over it's a hole desktop app witch we can all create programs and ideas for but its a little buggy because of old cocos2dx engine so you wont need the this code this will be better that's going to be a big project so please join in and contribute to it Ill try get a vid of that up too.

**Reply by paulocoutinhox** [reply]
Nice. I want more the joystick part. Thanks.

**Reply by GamesStudioTwo** [reply]
Much More deeper dive to AXMOL long video.
https://www.youtube.com/watch?v=saTY9peU9IY

Quick update I sorted the Mass problem I accidental said GetMass() instead of SetMass() so got the same mass as set same mass lol.

**Reply by paulocoutinhox** [reply]
Nice video. It is pure Axmol or you make something over Axmol?

**Reply by GamesStudioTwo** [reply]
Pure Axmol, just a custom TXML map reader I wrote I way back because cocos2dx was missing some parts but axmol. Cocos was missing Text and Animation and a other stuff. but axmol guys have have fixed the map reader to incorporate I think everything so my map reader is a bit redundant now. I could just use FastTXML  but i'm using a sprite sheet instead of tiles so it only creates 1 layer that's why only one draw call.

**Reply by paulocoutinhox** [reply]
You can make a pr to increase FastTXML compatibility and support for editor things.

**Reply by GamesStudioTwo** [reply]
sorry what's a PR

**Reply by paulocoutinhox** [reply]
Pull-Request.

You can submit your changes to github axmol project to add your features.

**Comment by MinhMeo412** [comment]
Hi! 
I'm new to Axmol. Could I see the joystick part with camera?

**Comment by GamesStudioTwo** [comment]
Hi I have not done much for a while but I'm about to start a new series on
the whol engine soon and program some simple games to get people going in
the right direction. So keep checking back i I will be giving away code too
like the joysticks and stuff
